src/helpers.js

```diff
@@ -250,23 +251,10 @@ export function getFormatedDateTime (timestamp) {
   }
 }
 
-export function formatDateFromNow (d, msg) {
-  const now = Date.now()
-  const date = new Date(d)
-
-  const suffixes = [ msg.secondsShort(), msg.minutesShort(), msg.hoursShort(), msg.daysShort(), msg.monthsShort(), msg.yearsShort() ]
-  const divitions = [ 1000, 60, 60, 24, 30, 12 ]
-  let elapsed = (now - date.getTime())
-  let suffix = ''
-
-  let currentIndex = 0
-  do {
-    suffix = suffixes[currentIndex]
-    elapsed = Math.floor(elapsed / divitions[currentIndex])
-    currentIndex++
-  } while (divitions[currentIndex] <= elapsed && currentIndex < suffixes.length)
-
-  return msg.timeAgo({ time: `${elapsed}${suffix}` })
+export function formatDateFromNow (date, msg, now = new Date()) {
+  // 'now' parameter allows reliable testing
+  // uses standard moment.relativeTimeThreshold values
+  return moment.duration(moment(date).diff(moment(now))).humanize(true)
```

Comment:
Why not just use `formatUptimeDuration` in `src/utils/format.js`?  Then the uptime string and the age of snapshot would all follow the same format on the VM Details page.

Reply:
My assumption was that we want to keep single number "humanized" format.

Reply:
Historically, there was a bunch of debate over the format of the text for how long ago type of things.  The debate around the uptime one was annoyingly long.

So at this point, as long as `humanize()` has localized text provided by moment, then we're good.  If not, we'll need to drop back to the `moment-duration-format` and our own localization strings.

Reply:
@rszwajko , @sgratch , @hstastna - As long as the humanized string are translated, I don't mind using it instead of a more uptime string format.